Articles/ShortArticles/TheDeveloperCertificateOfOrigin.md

```diff
@@ -0,0 +1,147 @@
+While CLAs have a number of advantages for a project from a legal standpoint (such as allowing for later re-licensing), they are unpopular with some developers and developer communities and are seen by many as an impediment to accepting contributions<sup>[3],[5]</sup>.
+
+To address the problems with CLAs and other similar appraoches, the Linux Foundation introduced the Developer Certificate of Origin (DCO) in 2004<sup>[6]</sup>.
+The DCO is a lightweight approach where developers "sign" every Git commit with a `Signed-off-by` line in the commit message:
+
+```
+This is the commit message
+
+Signed-off-by: First M. Last {first.last@someurl.org}
+```
+
+which implies that the developer is asserting the terms of the DCO<sup>[7]</sup> which, as of Version 1.1, states:
+
+> By making a contribution to this project, I certify that:
+> 
+> (a) The contribution was created in whole or in part by me and I
+>     have the right to submit it under the open source license
+>     indicated in the file; or
+> 
+> (b) The contribution is based upon previous work that, to the best
+>     of my knowledge, is covered under an appropriate open source
+>     license and I have the right under that license to submit that
+>     work with modifications, whether created in whole or in part
+>     by me, under the same open source license (unless I am
+>     permitted to submit under a different license), as indicated
+>     in the file; or
+> 
+> (c) The contribution was provided directly to me by some other
+>     person who certified (a), (b) or (c) and I have not modified
+>     it.
+> 
+> (d) I understand and agree that this project and the contribution
+>     are public and that a record of the contribution (including all
+>     personal information I submit with it, including my sign-off) is
+>     maintained indefinitely and may be redistributed consistent with
+>     this project or the open source license(s) involved.
+
+This `Signed-of-by` line can be added to the Git commit log automatically using the standard `-s` option with `git commit -s` (using information already registered with Git locally).
+
+Some major projects are using the DCO with Git commit sign-offs for handling contributions including the Linux Kernel<sup>[8]</sup> and more recently the source code for GitLab itself <sup>[3],[4]</sup>.
+In addition, the Linux Foundation's CII Best Practices Badge Program<sup>[9]</sup> contains the Silver-level item:
+
+* The project SHOULD have a legal mechanism where all developers of non-trivial amounts of project software assert that they are legally authorized to make these contributions.
+The most common and easily-implemented approach for doing this is by using a Developer Certificate of Origin (DCO)<sup>[7]</sup>, where users add "signed-off-by" in their commits and the project links to the DCO website.
+However, this MAY be implemented as a Contributor License Agreement (CLA), or other legal mechanism. {Met URL} <sup>[10]</sup>
```

Comment:
Can we remove "{Met URL}" in line 63?  I don't think it is relevant here. We should just end the quotation at the end of the sentence.

Reply:
Should be addressed in new commit badd685f8f5bf9f11a7a345e495083b25ecd127e